cmd/sonobuoy/app/results.go

```diff
@@ -134,7 +134,7 @@ func result(input resultsInput) error {
 
 		// Load file with a new reader since we can't assume this reader has rewind
 		// capabilities.
-		r, cleanup, err := getReader(input.archive)
+		r, cleanup, _ := getReader(input.archive) // TODO: handle error checking
 		defer cleanup()
```

Comment:
yeah this was marked as "unused variable `err`"

Reply:
I see, yeah I can agree with that. I'm a little worried that this is potentially masking errors because "only the last error is printed", but that's perhaps a separate discussion at a different time.